web-ui/src/pages/MainGame/MainGameScreen.tsx

```diff
@@ -55,33 +66,56 @@ export default class MainGameScreen extends React.Component<{}, GameState> {
     contract: null,
   };
 
-  componentDidUpdate(): void {
+  componentDidMount(): void {
+    this.props.registerOnMessageReceivedCallback(
+      getGameTopic(this.props.gameId, this.props.username),
+      MESSAGE_TYPE.ROUND_STARTED,
+      ({ playerCards }) => this.updatePlayerCards(playerCards),
```

Comment:
It's as easy as that to retrieve the cards, the json is already formatted correctly ^^